src/flowTypes.ts

```diff
@@ -396,7 +397,7 @@ export interface SendWhatsAppMsg extends Action {
   messageType: WhatsAppMessageType;
   header_type: WhatsAppHeaderType;
   header_text: string;
-  attachment: string;
+  attachment: Attachment;
```

Comment:
Is mailroom supporting this change? I believe that it is expecting a single URL as string, not an object